locale/core.yml

```diff
@@ -331,9 +331,9 @@ core:
     # These strings are displayed as error messages.
     error:
       generic_message: "Oops! Da ist was schief gelaufen. Bitte die Seite neu laden oder noch mal versuchen."
-      not_found_message: Leider nichts gefunen.
+      not_found_message: Leider nichts gefunden.
       permission_denied_message: Verboten.
-      rate_limit_exceeded_message: Nicht so hastig. Bitte gleich noch mal versuchen.
+      rate_limit_exceeded_message: Bitte warte einen Moment.
```

Comment:
Ich glaube das ist dann doch etwas zu verkürzt oder?. Im englischen steht: "You're going a little too quickly. Please try again in a few seconds."